src/ReactDOM.rei

```diff
@@ -478,6 +478,51 @@ external unmountComponentAtNode: Dom.element => unit =
 [@mel.module "react-dom"]
 external flushSync: (unit => unit) => unit = "flushSync";
 
+module Experimental: {
+  /*
+     preinitModule and preloadModule options.
+     https://react.dev/reference/react-dom/preinitModule#parameters
+   */
+  [@deriving jsProperties]
+  type preOptions = {
+    /* a required string. It must be 'script'. */
+    [@mel.as "as"] [@mel.optional]
+    _as: option([ | `script]),
+    /*
+        a required string. It must be "anonymous", "use-credentials", and "".
+        https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin
+     */
+    [@mel.optional]
+    crossOrigin: option(string),
+    /*
+        a string. A cryptographic hash of the module, to verify its authenticity.
+        https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity
+     */
+    [@mel.optional]
+    integrity: option(string),
+    /*
+        a string. A cryptographic nonce to allow the module when using a strict Content Security Policy.
+        https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/nonce
+     */
+    [@mel.optional]
+    nonce: option(string),
+  };
+
+  /* This module is used to bind to APIs for future versions of ReactDOM. There is no guarantee of backwards compatibility or stability. */
```

Comment:
Can we keep this comment below `module Experimental = ...`?

Reply:
Sure @davesnx. Updated yet.